tests/e2e/e2e_common.go

```diff
@@ -82,7 +82,7 @@ const (
 	podContainerCreatingState                  = "ContainerCreating"
 	poll                                       = 2 * time.Second
 	pollTimeout                                = 5 * time.Minute
-	pollTimeoutShort                           = 1 * time.Minute / 2
+	pollTimeoutShort                           = 1 * time.Minute
```

Comment:
This will increase the running time of e2e test cases by 19 minutes.
If 1 minute pollTimeout is needed for CnsRegisterVolume, then declare a new variable. Changing the exisiting variable will affect all the other test cases